cohesity/src/agents/special/agent_cohesity

```diff
@@ -70,8 +70,8 @@ def get_url(url):
             API_URL + url,
             headers=HEADER,
             verify=False)
-    except APIException as exp:
-        print(f"Cannot get Clusterstatus {exp}")
```

Comment:
Same here

Reply:
Same like above. Where is this exception defined?